tests/pattern-matching.lisp

```diff
@@ -196,16 +196,12 @@
            "Allows bindings of the form (:exact sym) or (:bind sym). The former requires EQL between sym and the actual car and returns no bindings. The latter binds the car to the given symbol and never fails to match."
            (ecase (car binding-car)
              (:exact
-              (if (eql (cadr binding-car) actual-car)
-                  nil
-                  'no-match))
+              (unless (eql (cadr binding-car) actual-car)
+                'no-match))
              (:bind
-                 ;; emacs is being weird about the indentation here, something specific about :bind
-                 `((,(cadr binding-car) . ,actual-car)))))
-         (car-match-multiple (binding-car actual-car)
```

Comment:
the "label" `car-match-multiple` was removed because it was not used (and sbcl was complaining)